campaigns/private-browsing/logo-swap.html

```diff
@@ -0,0 +1,203 @@
+  {% if blockable %}
+    <button class="block-snippet-button block-snippet-button-{{ snippet_id }}">
+    </button>
+  {% endif %}
+  {% if blockable or clickable %}
+    <div id="block-snippet-overlay-{{ snippet_id }}" class="block-snippet-overlay-{{ snippet_id }}">
+    </div>
+  {% endif %}
+  {% if fallback_url %}<a href="{{ fallback_url|safe }}" class="launch-pb">{% endif %}
+    <img class="icon" src="{{ icon }}" />
+  {% if fallback_url %}</a>{% endif %}
+  <p>{{ text|safe }}</p>
+</div>
+<script>
+//<![CDATA[
+(function() {
+    var snippet = document.getElementById('snippet-{{ snippet_id }}');
+    var newLogo = '{{ replacement_logo }}';
+    snippet.addEventListener('show_snippet', function(e) {
+        var brandLogo = document.getElementById('brandLogo');
+        if (brandLogo.src !== undefined) {
+            brandLogo.src = newLogo;
+        } else {
+            brandLogo.style.backgroundImage = 'url(' + newLogo + ')';
+        }
+        brandLogo.classList.add('switch-{{ snippet_id }}');
+
+        {% if clickable or blockable %}
+            var overlay = document.getElementById('block-snippet-overlay-{{ snippet_id }}');
+        {% endif %}
+
+        {% if clickable %}
+          snippet.addEventListener('mouseover', function(e) {
+            overlay.classList.add('block-snippet-overlay-glow-{{ snippet_id }}');
+          });
+          snippet.addEventListener('mouseout', function(e) {
+            overlay.classList.remove('block-snippet-overlay-glow-{{ snippet_id }}');
+          });
+          snippet.addEventListener('click', function () {
+              var links = snippet.querySelectorAll('a');
+              if (links.length) {
+                  links[0].click();
+              }
+          });
+        {% endif %}
+
+        {% if blockable %}
+          brandLogo.insertAdjacentHTML(
+            'afterbegin', '<button class="block-snippet-button block-snippet-button-{{ snippet_id }}"></button>');
+
+          var xbuttons = document.querySelectorAll('.block-snippet-button-{{ snippet_id }}');
+          var mouseover = function(event) {
+              overlay.classList.add('block-snippet-overlay-close-{{ snippet_id }}');
+              brandLogo.style.opacity = 0.6;
+          };
+          var mouseout = function(event) {
+              overlay.classList.remove('block-snippet-overlay-close-{{ snippet_id }}');
+              brandLogo.style.opacity = 1;
+          };
+          for (var k = 0, l = xbuttons.length; k < l; k++) {
+              var xbutton = xbuttons[k];
+              xbutton.addEventListener('mouseover', mouseover, false);
+              xbutton.addEventListener('mouseout', mouseout, false);
+          }
+        {% endif %}
+
+        var buttons = snippet.getElementsByClassName('launch-pb');
+
+        isTargetAvailable('privateWindow', function targetAvailable () {
+          sendMetric('impression-pb-available');
+          for (var i=0; i < buttons.length; i++) {
```

Comment:
Similar microoptimization as in https://github.com/mozilla/snippets/pull/56/files#r42121205